packages/devtools_app/lib/src/debugger/debugger_screen.dart

```diff
@@ -114,6 +114,13 @@ class DebuggerScreenBodyState extends State<DebuggerScreenBody>
         return ValueListenableBuilder(
           valueListenable: controller.currentParsedScript,
           builder: (context, parsedScript, _) {
+            if (scriptRef != null &&
+                parsedScript != null &&
+                !_shownFirstScript) {
+              ga.timeEnd(DebuggerScreen.id, analytics_constants.pageReady);
+              // TODO(annagrin): mark end of IPL timing for debugger page here.
```

Comment:
Fyi @annagrin to dispatch an event dwds can listen for as well.

Reply:
I am adding `ext.dwds.sendEvent` service extension to dwds, it will expect an event `{ 'type': 'DevtoolsReady', 'payload': {} }` .

See https://github.com/dart-lang/webdev/pull/1404